nemo/collections/asr/models/configs/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo.collections.asr.models.configs.ctc_models_config import (
+    AudioToMelSpectrogramPreprocessorConfig,
+    ConvASRDecoderConfig,
+    ConvASREncoderConfig,
+    EncDecCTCConfig,
+    EncDecCTCDatasetConfig,
+    EncDecCTCModelConfig,
+    JasperEncoderConfig,
+    SpecAugmentConfig,
+)
```

Comment:
What are the advantages of having:
* a separate file with model configs VS
* specific model config placed in the same file along with the given model

IMO the latter is cleaner, as params as "closer" to model, and whenever you write "from fileX import ModelX" you can change that to "from fileX import ModelX, ModelXConfig".

What do you think?

Reply:
Model configs are generally very large, id always keep them entirely seperate from the code itself. We can perform a similar `from fileX import ModelX, ModelXConfig` simply by importing the appropriate config inside the appropriate Model file (with a from import) so that it is accessible to users via the same syntax.

Reply:
It does not look right to me to have things like: ``JasperEncoderConfig``, ``AudioToMelSpectrogramPreprocessorConfig`` under ``nemo.collections.asr.models.configs``.

Let's only have *model* configs under ``nemo.collections.asr.models.configs.*``
I would put *Module* configs in the same file with its module.